doc/sphinxman/source/sapt.rst

```diff
@@ -237,59 +248,90 @@ These requirements are imposed whenever a SAPT calculation is requested
 but can also be set explicitly with the ``no_reorient`` and ``symmetry
 c1`` molecule keywords, as in the example above. As a reminder, only
 SAPT0 can handle the interaction of both closed- and open-shell monomers.
-Higher-order SAPT is only available for computation of interactions between
-closed-shell singlets.
+Higher-order SAPT and SAPT(DFT) are currently available only 
+for computation of interactions between
+closed-shell singlets.  The SAPT codes in |PSIfour| have been written
+to utilize density fitting, which is much faster than using conventional
+4-index electron repulsion integrals.  This happens automatically and
+does not require any additional keywords to be specified (both the 
+SAPT computations and the underlying Hartree-Fock computations will
+utilize density fitting).
+
+
+For SAPT(DFT), the user will need to manually specify the GRAC shift of 
+both monomers, defined by the difference of ionization potential (IP) and
+Kohn-Sham HOMO energy. The ionization potential data for many common molecules 
+is available in NIST Chemistry Webbook. Alternatively, one can estimate the
+ionization potential of molecule by computing the energy difference between
+the molecule as given, and the molecule after one electron has been removed
+(e.g., the energy difference between a neutral molecule and its cation).
+
+The values of GRAC shifts should be given in Hartree. For the example above,
+the GRAC shift value of both molecules are calculated to be 0.1307, and the 
+user would specify them using the following keywords::
```

Comment:
Hi all, just want to confirm that these edits to the examples are intentional.  The different df_basis_scf, etc., options used to be required long ago (apparently), but are no longer so.  Leaving them in here is confusing to a new user (this will all be set by default) so I'd like those lines removed.